contracts/BidderRegistry.sol

```diff
@@ -163,22 +163,28 @@ contract BidderRegistry is IBidderRegistry, Ownable, ReentrancyGuard {
      */
     function retrieveFunds(
         bytes32 commitmentDigest,
-        address payable provider
+        address payable provider,
+        uint256 residualBidAfterDecay
```

Comment:
It seems like this `residualBidAfterDecay` is a multiplier and not the residual bid amount itself? A comment or name change would help clarify